forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt

```diff
@@ -0,0 +1,15 @@
+Name:Patchplate Resolute
+ManaCost:3
+Types:Artifact Creature Soldier
+PT:3/3
+K:Unearth:1 W
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigBoon | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Any | Execute$ TrigBoon | Secondary$ True | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ SpellCast
+SVar:SpellCast:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | OneOff$ True | ActivationLimit$ 1 | TriggerZones$ Command | Execute$ ReplEffAddCounter | TriggerDescription$ When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
```

Comment:
Try without ActivationLimit, imo the OneOff should already get rid of it anyway?